services/worker/src/worker/dtos.py

```diff
@@ -136,7 +147,10 @@ class ConfigSize(TypedDict):
     num_columns: int
 
 
-class SplitSize(SplitItem):
+class SplitSize(TypedDict):
```

Comment:
Why to FullSplitItem instead of TypedDict?

Reply:
yes, it could be. But I found that a lot of indirections are not easy to follow. Easier to just put the four properties